### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by Serms1999**

### CODE REVIEW

File: Assignment4/Serms1999-22A305/Task07.py
Comment (anaigmo) [inline Assignment4/Serms1999-22A305/Task07.py:1]: missing rdflib alternatives for all exercises in this task

File: Assignment4/Serms1999-22A305/Task07.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# TO DO
+# Visualize the results
+
+from rdflib.plugins.sparql import prepareQuery
+NS = Namespace("http://somewhere#")
+
+q1 = prepareQuery('''
+    SELECT ?subclass WHERE {
+      ?subclass rdfs:subClassOf ns:LivingThing.
+    }
```
Comment (anaigmo) [inline Assignment4/Serms1999-22A305/Task07.py:27]: missing recursive subclass of relationships

File: Assignment4/Serms1999-22A305/Task07.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# Visualize the results
+from rdflib.namespace import FOAF
+
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0")
+
+q4 = prepareQuery('''
+    SELECT DISTINCT ?person
+    WHERE {
+       ?person foaf:knows ns:RockySmith .
+    }
```
Comment (anaigmo) [inline Assignment4/Serms1999-22A305/Task07.py:81]: list name, nor iri

**Comment by Serms1999** [comment]
I have just fixed the issues

**Comment by anaigmo** [comment]
Submitted and mostly fixed, no more PRs allowed

**Pull request merged**